### DISCUSSION TITLE
Printing table with more than 100 entries?

### DISCUSSION THREAD

**Discussion by JOJ0** [Textualize/rich#3164] [Q&A] [answered]
Is it generally possible to print a Rich table with more than 100 rows?

After learning and reading docs I thought I understood that my panels which reside in a layout probably prevent the viewing of more than my terminal is capable to display, simply because of a panel/layout is limited by the terminal size. Makes sense so far.

Should it be possible now that a plain `rich.table.Table()` constructed table displays more than 100 rows or is there a hardcoded limit? I'm a little afraid of going Textualize... ;-) 

Thanks, can certainly provide further details and code snippets if there's no simple answer to that :-)

Great, great library BTW!

**Comment by willmcgugan** [comment, marked as answer]
There's no limit to the size of a table you can construct. But layout is limited to the size of the terminal. Make the leap to [Textual](https://textual.textualize.io/widgets/data_table/#__tabbed_1_2) which I think is even easier, and doesn't suffer from this issue.

**Reply by JOJ0** [reply]
I'm sorry my bad - I had a bug in my "data retrieval code" capping at 100 :-/

Thanks for the quick reply though! Appreciated!

For now I'm good with just using panel + table :-)
